# File: DiffData.msg
float32 time_difference

